rest/v10/interactions.ts

```diff
@@ -71,7 +71,9 @@ export type RESTPostAPIApplicationCommandsResult = APIApplicationCommand;
 /**
  * https://discord.com/developers/docs/interactions/application-commands#edit-global-application-command
  */
-export type RESTPatchAPIApplicationCommandJSONBody = StrictPartial<RESTPostAPIApplicationCommandsJSONBody>;
+export type RESTPatchAPIApplicationCommandJSONBody = StrictPartial<
+	Omit<RESTPostAPIApplicationCommandsJSONBody, 'dm_permission'>
+>;
```

Comment:
Did the wrong one here, should be all the guild endpoints - post, patch, and put